File: Android/NewApp/app/src/main/java/com/tanhoanngoc/glofttest/newapp/Fruit.java
package com.tanhoanngoc.glofttest.newapp;

public class Fruit {
    private String name;
    private String description;
    private int igmID;

    public Fruit(String name, String description, int igmID) {
        this.name = name;
        this.description = description;
        this.igmID = igmID;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getIgmID() {
        return igmID;
    }

    public void setIgmID(int igmID) {
        this.igmID = igmID;
    }
}

File: Android/MyApplication/app/src/main/java/com/tanhoanngoc/glofttest/myapplication/Fruit.kt
package com.tanhoanngoc.glofttest.myapplication

class Fruit {

    private var name : String = ""
    private var description : String = ""
    private var pictureID : Int = -1

    constructor(name: String, description: String, pictureID: Int) {
        this.name = name
        this.description = description
        this.pictureID = pictureID
    }

    public fun getName() : String{
        return name;
    }

    public fun getDescription() : String {
        return description;
    }

    public fun getPictureId() : Int {
        return pictureID;
    }
}